Application/Api/Education/Certificate/Generator/Repository/ESBD/EsbdGymKurshalbjahreszeugnis.php

```diff
@@ -0,0 +1,679 @@
+                ->styleBorderBottom($isGradeUnderlined ? '1px' : '0px', '#000')
+                ->styleMarginTop($marginTop)
+                , '12%');
+    }
+
+    /**
+     * @param string $MarginTop
+     *
+     * @return Slice
+     */
+    private function setPointsOverview($MarginTop = '20px')
+    {
+
+        $textSize = '10px';
+        $slice = new Slice();
+        $slice
+            ->addSection((new Section())
+                ->addElementColumn((new Element())
+                    ->setContent('Für die Umsetzung der Noten in Punkte gilt:')
+                    ->styleTextSize($textSize)
+                )
+            );
+
+        $section = new Section();
+        $this->setColumnElement($section, 'Notenstufen', $textSize);
+        $this->setColumnElement($section, 'sehr gut', $textSize);
+        $this->setColumnElement($section, 'gut', $textSize);
+        $this->setColumnElement($section, 'befriedigend', $textSize);
+        $this->setColumnElement($section, 'ausreichend', $textSize);
+        $this->setColumnElement($section, 'mangelhaft', $textSize);
+        $this->setColumnElement($section, 'ungenügend', $textSize, false, true);
+        $slice
+            ->addSection($section);
+
+        $section = new Section();
+        $this->setColumnElement($section, 'Noten', $textSize);
+        $this->setColumnElement($section, '+&nbsp;&nbsp;&nbsp;1&nbsp;&nbsp;&nbsp;-', $textSize);
+        $this->setColumnElement($section, '+&nbsp;&nbsp;&nbsp;2&nbsp;&nbsp;&nbsp;-', $textSize);
+        $this->setColumnElement($section, '+&nbsp;&nbsp;&nbsp;3&nbsp;&nbsp;&nbsp;-', $textSize);
+        $this->setColumnElement($section, '+&nbsp;&nbsp;&nbsp;4&nbsp;&nbsp;&nbsp;-', $textSize);
+        $this->setColumnElement($section, '+&nbsp;&nbsp;&nbsp;5&nbsp;&nbsp;&nbsp;-', $textSize);
+        $this->setColumnElement($section, '6', $textSize, false, true);
+        $slice
+            ->addSection($section);
+
+        $section = new Section();
+        $this->setColumnElement($section, 'Punkte', $textSize, true);
+        $this->setColumnElement($section, '15 14 13', $textSize, true);
+        $this->setColumnElement($section, '12 11 10', $textSize, true);
+        $this->setColumnElement($section, '09 08 07', $textSize, true);
+        $this->setColumnElement($section, '06 05 04', $textSize, true);
+        $this->setColumnElement($section, '03 02 01', $textSize, true);
+        $this->setColumnElement($section, '00', $textSize, true, true);
+        $slice
+            ->addSection($section);
+
+        return $slice
+            ->styleMarginTop($MarginTop);
+    }
+
+    /**
+     * @param Section $section
+     * @param string $name
+     * @param $textSize
+     * @param bool $isBorderBottom
+     * @param bool $isBorderRight
+     */
+    private function setColumnElement(
+        Section $section,
+        $name,
+        $textSize,
+        $isBorderBottom = false,
+        $isBorderRight = false
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$isBorderRight" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)